Cargo.toml

```diff
@@ -77,9 +77,6 @@ reqwest = { version = "0.12.20", default-features = false, features = [
 # (And cargo doesn't let us have platform specific lints here)
 rust.unsafe_code = "deny"
 
-# Intentional break from the lint set. Intended to be temporary
-rust.unexpected_cfgs = { level = "warn", check-cfg = ['cfg(FALSE)'] }
```

Comment:
We should restore this back in the lints below; it's in the lint set but not in this file.

Otherwise, if it's already warn by default, then we should remove it from the lint set entirely.

Reply:
In fact, I'd like to treat getting an answer here as blocking. If it's the case that this is warn by default, feel free to dismiss this review.